src/openms/source/FILTERING/CALIBRATION/InternalCalibration.cpp

```diff
@@ -61,11 +61,12 @@ namespace OpenMS
     {
       for (Size i = 0; i < pcs.size(); ++i)
       {
+				pcs[i].setMetaValue("mz_raw", pcs[i].getMZ());
```

Comment:
please augment the InternalCalibration_test.cpp and check if the value is really annotated